src/Handler/StreamHandler.php

```diff
@@ -94,6 +94,10 @@ private function invokeStats(
         }
     }
 
+    /**
+     * @param resource $stream
+     * @param mixed $startTime
```

Comment:
what types can this be?

Reply:
I am resolving this.. 
Please check my previous answer and let's continue the discussion there in order to avoid having the same discussion to multiple places :)